app/templates/_profile_prod.gradle

```diff
@@ -30,6 +30,8 @@ task setProdProperties(dependsOn: bootRun) << {
     }
 }
 
+setProdProperties.dependsOn 'gulp_ngconstant:prod'
```

Comment:
Yes you're right it is not required to execute explicitly ``ngconstant:prod`` here.